qutebrowser/mainwindow/statusbar/percentage.py

```diff
@@ -34,6 +35,7 @@ def __init__(self, parent=None):
         self.set_perc(0, 0)
         self.raw = False
 
+    @throttle(100)
```

Comment:
Is the pyqtSlot decorator useful if it's wrapped in Python wrapper object anyway?